### DISCUSSION TITLE
Is Jackson support important to you?

### DISCUSSION THREAD

**Discussion by larsga** [schibsted/jslt#185] [Q&A]
Some users need to be able to [use JSLT without Jackson](https://github.com/schibsted/jslt/issues/123), and to support these users [work is underway](https://github.com/schibsted/jslt/tree/own-json-interfaces) on a JSLT implementation that has no dependencies on Jackson.

It will be possible to support transforming Jackson `JsonNode` objects into new `JsonNode` objects, but performance will likely suffer somewhat.

Are there any users for whom this matters because they really do need Jackson?

**Comment by allquixotic** [comment]
I'm actually mostly interested in using jslt for scripting in the CLI or with scripting languages that don't have their own (good) JSON libraries already, or where getting those libraries onto the system would be a lot of trouble. For that reason, I'm mostly interested in the CLI mode of jslt, not the library API calls. For my use, I would be fine without jackson support (and in fact, the lighter the dependencies, the better, probably).
